src/Main.java

```diff
@@ -1,5 +1,106 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+        System.out.println("Объявление массивов");
+        int [] one = new int[]{1, 2, 3};
```

Comment:
здесь предполагалось создать массив длины 3 и потом заполнить его значениями, как в подсказке с 12ю:
```suggestion
        int [] one = new int[3];
        for (int i = 0; i < one.length; i++) {
            one[i] = i + 1;
        }
```

Reply:
потому что способ создания `int [] one = new int[]{1, 2, 3}` равносилен такому: `int [] one = {1, 2, 3}`